packages/models-library/src/models_library/projects.py

```diff
@@ -87,6 +88,14 @@ class ProjectAtDB(BaseProjectModel):
         False, description="Defines if a study is available publicly"
     )
 
+    sharing_networks: SharingNetworks = Field(
```

Comment:
does this really need to be inside the project?

Reply:
After giving it a bit more though I can also concur that it should stay outside of the `projects`. Will create a dedicated table for it.